web/app/js/components/util/Progress.jsx

```diff
@@ -0,0 +1,33 @@
+import LinearProgress from '@material-ui/core/LinearProgress';
+import { withStyles } from '@material-ui/core/styles';
+
+const colorLookup = {
+  good: {
+    colorPrimary: '#c8e6c9', // background bar color (lighter)
```

Comment:
per https://material-ui.com/style/color/, `#c8e6c9` is documented as `green[100]`:

```js
import red from '@material-ui/core/colors/green';
...
colorPrimary: green[100], // background bar color (lighter)
```

is it advisable/appropriate to reference them this way?